src/test/java/net/minecraftforge/debug/ModelLoaderRegistryDebug.java

```diff
@@ -48,24 +39,35 @@
 import net.minecraftforge.common.property.IExtendedBlockState;
 import net.minecraftforge.common.property.IUnlistedProperty;
 import net.minecraftforge.common.property.Properties;
-import net.minecraftforge.fml.common.FMLLog;
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.common.Mod.EventHandler;
 import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
 import net.minecraftforge.fml.common.registry.GameRegistry;
 import net.minecraftforge.fml.relauncher.Side;
+import org.apache.logging.log4j.Logger;
 
-import com.google.common.collect.Lists;
+import javax.vecmath.AxisAngle4d;
+import javax.vecmath.Matrix4f;
+import javax.vecmath.Quat4f;
+import javax.vecmath.Vector3d;
+import javax.vecmath.Vector4f;
+import java.util.ArrayList;
+import java.util.List;
 
 @Mod(modid = ModelLoaderRegistryDebug.MODID, name = "ForgeDebugModelLoaderRegistry", version = ModelLoaderRegistryDebug.VERSION, acceptableRemoteVersions = "*")
 public class ModelLoaderRegistryDebug
 {
+    public static final boolean ENABLED = false;
     public static final String MODID = "forgedebugmodelloaderregistry";
     public static final String VERSION = "1.0";
+    private static Logger logger;
```

Comment:
In different test mods, the logger fields have their name in different cases. You need to unify them.